src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php

```diff
@@ -29,7 +29,7 @@ class <?= $class_name ?> extends AbstractController
 {
     use ResetPasswordControllerTrait;
 
-    private $resetPasswordHelper;
+    private<?= $is_typed ? ' ResetPasswordHelperInterface' : null ?> $resetPasswordHelper;
```

Comment:
Example usage of the global `$use_typed_properties` variable that is available in all templates and should be used anytime we are determining if typed properties should be used in all templates.

```suggestion
    private <?= $use_typed_properties ? 'ResetPasswordHelperInterface ' : null ?>$resetPasswordHelper;
```